pkg/openthread/doc.txt

```diff
@@ -4,4 +4,7 @@
  * @ingroup  net
  * @brief    Provides a RIOT adaption of the OpenThread network stack
  * @see      https://github.com/openthread/openthread
- */
\ No newline at end of file
+ * @note     Currently there's only support for @ref drivers_at86rf2xx.
+ *           There's work in progress 
```

Comment:
There's a work in progress to support more radio drivers, see [this issue](https://github.com/RIOT-OS/RIOT/issues/10045)
... split in 80 columns :)